src/components/ListTitle.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement } from 'react'
+import { SvgProps } from 'react-native-svg'
+import styled from 'styled-components/native'
+
+const Container = styled.View`
+  min-height: 54px;
+  align-items: center;
+  margin-bottom: 32px;
+  margin-top: 32px;
+`
+
+const ScreenTitle = styled.Text`
+  line-height: 18px;
+  text-align: center;
+  font-size: ${props => props.theme.fonts.headingFontSize};
+  color: ${props => props.theme.colors.lunesGreyDark};
+  font-family: ${props => props.theme.fonts.contentFontBold};
+`
+
+const ScreenSubTitle = styled.Text`
+  margin-top: 25px;
+  text-align: center;
+  font-size: ${props => props.theme.fonts.defaultFontSize};
+  color: ${prop => prop.theme.colors.lunesGreyDark};
+  font-family: ${props => props.theme.fonts.contentFontBold};
```

Comment:
`text-align, font-family...`, which are same for all children of `Container` (Title, Subtitle, Description) can be moved to `Container`.